src/Database.pas

```diff
@@ -2911,7 +2913,7 @@ procedure CreateUnit(p, mix: integer);
       Status := 0;
       SavedStatus := 0;
       Inc(Model[mix].Built);
-      Home := -1;
+      Home := 0; //Lagi: -1 is homeless, support free settler
```

Comment:
I'd also have to say that this change is actually interesting in some ways.
The positive thing is that it reduces micro-management a bit.
The downside is that starting expansion tactics must change, and that the city with +1 food in your commit will have some strange long-lasting bonus. The `freeciv` project chose to introduce a small killing factor to homeless settlers, so that the starting settler takes N damage points per turn (eventually dying or building a city).